src/btree/bt_cursor.c

```diff
@@ -1050,32 +1037,20 @@ __wt_btcur_search_near(WT_CURSOR_BTREE *cbt, int *exactp)
      */
     if (valid) {
         exact = cbt->compare;
```

Comment:
This is the only place where the deletion wasn't straightforward, it requires additional review.